### DISCUSSION TITLE
Windows 11 - Non Enterprise Licensing

### DISCUSSION THREAD

**Discussion by Burns-Sleep** [simeononsecurity/Windows-Optimize-Harden-Debloat#38] [Q&A] [answered]
Hello! 

Great work with this script! I learned a lot of new things about Windows privacy from reading through it. I can't imagine the amount of time and effort that was thrown into this project.

I plan on performing a completely new install of windows 11 Professional in the near future. I know that since I won't be using enterprise licensing, many features will not be available to me. 

Is this script still usable without those? I know I can comment out the lines targeting those features but wanted to know if that is still recommended for people in my situation.

Thank you!

**Comment by simeononsecurity** [comment, marked as answer]
Yes, you can still use the script there. However you'll miss out on all the enterprise specific features.
This includes some of the better windows defender settings, the ability to permanently disable windows telemetry, etc.
Additionally, you shouldn't have to modify the script in anyway. It'll set the settings, but they will only work for enterprise versions.
